add-column.md

```diff
@@ -18,6 +18,121 @@ The user must have the `CREATE` [privilege](privileges.html) on the table.
+~~~ 
+
+~~~ sql
+> SHOW COLUMNS FROM accounts;
+~~~ 
+
+~~~
++-----------+-------------------+-------+---------+-----------+
+|   Field   |       Type        | Null  | Default |  Indices  |
++-----------+-------------------+-------+---------+-----------+
+| id        | INT               | false | NULL    | {primary} |
+| balance   | DECIMAL           | true  | NULL    | {}        |
+| names     | STRING            | true  | NULL    | {}        |
++-----------+-------------------+-------+---------+-----------+
+~~~
+
+
+### Add Multiple Columns
+
+~~~ sql
+> ALTER TABLE accounts ADD COLUMN location STRING, ADD COLUMN amount DECIMAL;
+~~~
+
+~~~ sql
+> SHOW COLUMNS FROM accounts;
+~~~ 
+
+~~~
++-----------+-------------------+-------+---------+-----------+
+|   Field   |       Type        | Null  | Default |  Indices  |
++-----------+-------------------+-------+---------+-----------+
+| id        | INT               | false | NULL    | {primary} |
+| balance   | DECIMAL           | true  | NULL    | {}        |
+| names     | STRING            | true  | NULL    | {}        |
+| location  | STRING            | true  | NULL    | {}        |
+| amount    | DECIMAL           | true  | NULL    | {}        |
++-----------+-------------------+-------+---------+-----------+
+
+~~~
+
+### Add a Non-Null Column with a Default Value
+
+~~~ sql
+> ALTER TABLE accounts ADD COLUMN interest DECIMAL NOT NULL DEFAULT (DECIMAL '1.3');
+~~~ 
+
+~~~ sql
+> SHOW COLUMNS FROM accounts;
+~~~ 
+
+~~~
++-----------+-------------------+-------+---------------------------+-----------+
+|   Field   |       Type        | Null  |          Default          |  Indices  |
++-----------+-------------------+-------+---------------------------+-----------+
+| id        | INT               | false | NULL                      | {primary} |
+| balance   | DECIMAL           | true  | NULL                      | {}        |
+| names     | STRING            | true  | NULL                      | {}        |
+| location  | STRING            | true  | NULL                      | {}        |
+| amount    | DECIMAL           | true  | NULL                      | {}        |
+| interest  | DECIMAL           | false | ('1.3':::STRING::DECIMAL) | {}        |
++-----------+-------------------+-------+---------------------------+-----------+
+~~~
+
+
+### Add a Non-Null Column with Unique Values
+
+~~~ sql
+> ALTER TABLE accounts ADD COLUMN cust_number DECIMAL UNIQUE NOT NULL;
+~~~ 
+ 
+
+### Add a Column with Collation
+
+~~~ sql
+> ALTER TABLE accounts ADD COLUMN more_names STRING COLLATE en;
+~~~ 
+ 
+
+### Add a Column to a Column Family
+
+~~~ sql
+# Add a Column to a Column Family
+> ALTER TABLE accounts ADD COLUMN location1 STRING CREATE FAMILY new_family;
+~~~ 
+
+
+~~~ sql
+# Add a Column to an Existing Column Family
+> ALTER TABLE accounts ADD COLUMN location2 STRING FAMILY existing_family;
+~~~
+
+### Add a Column Using CREATE IF NOT EXISTS FAMILY
```

Comment:
I would prefer keeping the example for the sake of completeness. Else that is the only case not covered in the document. But I can remove the explanation and keep it consistent with the other examples. Also, about pointing to the column families documentation, we have already stated that in the See Also section right after the examples. Should I restate it for emphasis? Let me know.